content/python/concepts/dictionaries/terms/clear/clear.md

```diff
@@ -0,0 +1,34 @@
+---
+Title: '.clear()'
+Description: 'Removes all entries in a dictionary.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Dictionaries'
+  - 'Methods'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The `.clear()` method of a Python dictionary removes all the entries in the dictionary. It takes no arguments.
+
+## Syntax
+
+```py
+dictionary.clear()
+```
+
+## Codebyte Example
+
+The following example creates a dictionary with some values, then replaces the contents by clearing the dictionary and adding different values.
+
+```codebyte/py
```

Comment:
Change codebytes to have the full name of the programming language as stated in the content standards.
```suggestion
```codebyte/python
```